src/wizards/tokenSwapDealWizard/stages/termsStage/termsStage.ts

```diff
@@ -53,4 +58,9 @@ export class TermsStage implements IBaseWizardStage {
   checkedForUnsavedChanges() {
     this.hasUnsavedChanges = this.termClauses.filter(viewModel => viewModel.viewMode === "edit").length > 0;
   }
+
+  private getDefaultTermsViewModes(wizardType: WizardType): EditingCard["viewMode"][] {
+    const isACreateWizard = [WizardType.createOpenProposal, WizardType.createPartneredDeal].includes(wizardType);
```

Comment:
Future:
Wonder if checks like this can go into WizardService, and if not, what could that class/collection be?
(currently I understand "Service" as a convenience class as well)

```ts
public isACreateWizard(wizardType): boolean {}
```